HW_8/Task_1.js

```diff
@@ -0,0 +1,71 @@
+// Task 1
+// Перед вами массив чисел [7, 8, 2, 30, 85, 95, 77, 94, 37, 31], используя методы массивов сделайте следующее:
+//   1. forEach - выведите в консоль все числа, делящиеся без остатка на 3 // 30
+
+const arrayTask1 = [7, 8, 2, 30, 85, 95, 77, 94, 37, 31];
+const canDivideByThree = [];
+
+arrayTask1.forEach((element) => {
+if (element % 3 === 0) {
+  canDivideByThree.push(element);
+ }
+});
+console.log(canDivideByThree)
+
+//   2. map - создайте новый массив, в котором из каждого элемента изначального массива вычли длину изначального массива 
+//      // [-3, -2, -8, 20, 75, 85, 67, 84, 27, 21]
+
+const arrayTask2 = [-3, -2, -8, 20, 75, 85, 67, 84, 27, 21];
+const arrMinus2 = arrayTask2.map((element) => {
+  return element - 2;
+});
```

Comment:
Откуда `2` взялось ? 
`вычли длину изначального массива`
```suggestion
const arrMinus2 = arrayTask2.map((element, i, originalArr) => {
  return element - ...;
});
```

Reply:
чет я сам забыл, почему я это сделал, видимо, когда копипастил, что-то нерарочно :(
Спасибо!